dash_bio/component_factory/_volcano.py

```diff
@@ -322,6 +368,10 @@ def figure(
     - highlight_color (string; default 'red'): Color of the data points
         highlighted because considered significant. Can be in any color
         format accepted by plotly.graph_objects.
+    - Additional keys (misc.): Arbitrary arguments can be passed to modify the
+        Layout and styling of the graph. A full reference of acceptable args is
+        available [here](https://plotly.com/python-api-reference/generated/plotly.graph_objects
+        .Layout.html).
```

Comment:
Since we're accepting `**kwargs` here and passing them to `go.Layout`, do we actually need _any_ of the other args you're adding here? We can list some of the common ones (including the ones you're including explicitly now) in the description here, they'd all work identically as part of kwargs.

`x_axis` and `y_axis`: in the docstring there's no `_` and I'd argue it's cleaner without, then these can just be left to kwargs as well.

In order to keep the defaults you have but allow users to update the layout arbitrarily with kwargs after that, I suspect the easiest would be to generate the default layout first, then do `layout.update(**kwargs)`. A little exploring shows that this does a deep merge, so if your kwargs are just `xaxis=dict(color="red")` you'll still get the rest of the x axis as set in the default layout.

Reply:
> do we actually need any of the other args you're adding here?

I thought it might be nice to have the most commonly used args available explicitly, especially for users that use IDE's which show parameter info, but simply including them as part of the documentation is also an option. However, retaining them as explicit args would also potentially save users a trip to the Plotly graphing library docs. I think it's a trade-off between cleaner params (with everything as kwargs) and easier "default" use of the component.

Reply:
Fair point re: IDE docs & completion. That said for everything except `height` and `width` you're still going to need to visit the graphing library docs to find out what goes inside the object, and it feels a bit arbitrary to be pulling in just a few layout options - as if the ones in kwargs are less-fully-supported or something. So I still think we'll be better off removing all of them and just documenting a few (the set you've proposed adding here looks good) in the docstring.

Reply:
> you're still going to need to visit the graphing library docs to find out what goes inside the object,

That's a great point - Updated in a5b996a.